service/internal/auth/authn.go

```diff
@@ -222,6 +223,13 @@ func (a Authentication) MuxHandler(handler http.Handler) http.Handler {
 			return
 		}
 
+		md, ok := metadata.FromIncomingContext(ctxWithJWK)
+		if !ok {
+			md = metadata.New(nil)
+		}
+		md.Append("access_token", ctxAuth.GetRawAccessTokenFromContext(ctxWithJWK, nil))
+		ctxWithJWK = metadata.NewIncomingContext(ctxWithJWK, md)
```

Comment:
This seems unnecessary now? Are you referring to it later?

Reply:
Honestly I'd rather punt and refactor wholly later when we are not under a time crunch.